dbus/dbus_libgio_test.go

```diff
@@ -35,3 +48,295 @@ func TestIsGUID(t *testing.T) {
 	guid := libgio.GenerateGUID()
 	assert.True(t, libgio.IsGUID(guid))
 }
+
+func TestBusGet(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
```

Comment:
Does it take that long?

Reply:
No, it was a way to disable tests that need a dbus daemon, we do that way in the backend (e.g. mongo-related tests).
Never mind, I'm removing the switch, as it doesn't make too much sense.